source-code/core/src/test/testConfigFile.ts

```diff
@@ -15,36 +16,40 @@ export async function testConfigFile(args: {
 	file: string
 	env: InlangEnvironment
 }): Promise<Result<true, TestConfigException>> {
-	try {
-		// BEGIN
-		// throws if an error occurs
-		importKeywordUsed(args.file)
-		// END
-		const { defineConfig } = await import("data:application/javascript;base64," + btoa(args.file))
-		const config = await defineConfig(args.env)
-		const [, exception] = await testConfig({ config })
-		if (exception) {
-			throw exception
-		}
-		return [true, undefined]
-	} catch (error) {
-		return [undefined, error as TestConfigException]
+	const [, importKeywordUsedException] = importKeywordUsed(args.file)
+	if (importKeywordUsedException) {
+		return [undefined, importKeywordUsedException]
 	}
+
+	const { defineConfig } = await import("data:application/javascript;base64," + btoa(args.file))
+	let config = await defineConfig(args.env)
+	config &&= await setupPlugins({ config, env: args.env })
```

Comment:
@samuelstroschein the functionality did not test plugins. I have added it and refactored the file to not use `throw`. Can you please review just this file and the tests?